src/cleaner/scripts/check_deleted_files.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import multiprocessing
+from cleaner.utils import common
+
+logger = multiprocessing.get_logger()
+
+
+def list_and_check_files(arg, log=logger):
+    files = common.run_cmd("source ./scripts/list_deleted_files.sh 2>/dev/null", log)
+    if len(files) <= 1:
+        log.info("no deleted files found.")
+        return
+    else:
+        # skip the field names from the command
+        files = files[1:]
+
+    for f in files:
+        f_fields = f.split(" ")
```

Comment:
Is output of the first column guaranteed not having space?

Reply:
Yes, the first column of the output is process id and it has no space.